Src/AutoFixture/Kernel/Seed.cs

```diff
@@ -1,145 +0,0 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Reflection;
-
-namespace Ploeh.AutoFixture.Kernel
-{
-    /// <summary>
-    /// Encapsulates a seed for a given type.
-    /// </summary>
-    public class Seed : ICustomAttributeProvider, IEquatable<Seed>

```

Comment:
This is an orphaned file. It looks like the file was renamed in 8c3fb3fa3ff23831b03f0648c7e37cec2746e1fe, but that the original file remained on disc.

I'll remove it from *master* shortly.